TicTacToeGame.cpp

```diff
@@ -0,0 +1,97 @@
+#include <iostream>
+#include <iomanip>
+
+void initializeBoard(int a[3][3]) {
+    for(int i = 0; i < 3; i++) {
+        for(int j = 0; j < 3; j++) {
+            a[i][j] = 0;
+        }
+    }
+}
+
+bool isOccupied(int row, int col, int a[3][3]) {
+    return a[row][col] != 0;
+}
+
+bool isOutOfBounds(int row, int col) {
+    return (row < 0 || row > 2 || col < 0 || col > 2);
+}
+
+bool isNotAvailable(int row, int col, int a[3][3]) {
+    return (isOutOfBounds(row, col) || isOccupied(row, col, a));
+}
+
+void enterASymbol(int a[3][3], int player) {
+    int row, col;
+    do{
+        if(player == 1) std::cout << "It's X turn! Please enter your coordinates: " << std::endl;
+        else std::cout << "It's O turn! Please enter your coordinates: " << std::endl;
+        std::cin >> row >> col;
+        row--;
+        col--;
+        if(!isNotAvailable(row, col, a)) {
+            a[row][col] = player;
+            break;
+        } else std::cout << "Sorry, but that space isn't available / doesn't exist! Try again! " << std::endl;
+    } while(isNotAvailable(row, col, a));
+}
+
+char symbol(int a) {
+    if(a == 1) return 'X';
+    else if(a == -1) return 'O';
+    else return ' ';
+}
+
+void printOutBoard(int a[3][3]) {
+        std::cout << "     1   2   3 " << std::endl;
```

Comment:
Before printing the board clear the console
Use these two cool lines

    std::cout << "\033[;H"; // Moves cursor to the top left
    std::cout << "\033[J"; // Clears the console